prow/cmd/deck/main.go

```diff
@@ -815,6 +816,19 @@ func handleFavicon(staticFilesLocation string, ca jobs.ConfigAgent) http.Handler
 	}
 }
 
+func handleSpyglass(spyglass bool) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		setHeadersNoCaching(w)
+		b, err := json.Marshal(spyglass)
+		if err != nil {
+			logrus.WithError(err).Error("Error marshalling spyglass config.")
+			http.Error(w, "Failed to marshal spyglass config.", http.StatusInternalServerError)
+			return
+		}
+		fmt.Fprintf(w, "var spyglass = %s;", b)
```

Comment:
To match the other endpoints, we should respect the `var` query parameter instead of hard coding to `spyglass`.